usr/share/rear/lib/global-functions.sh

```diff
@@ -622,7 +622,8 @@ umount_mountpoint() {
     Log "Unmounting '$mountpoint'"
     umount $v $mountpoint >&2
     if [[ $? -eq 0 ]] ; then
-        return 0
+        echo 0
+	return
```

Comment:
From plain looking at the code here I think in general
all 'is_something' testing functions
should not provide the result as a string on stdout
but as usual via the exit code return value so that a testing function
could be used as usual by calling
```
is_something && ...
is_something || ...
```